internal/tofu/eval_variable.go

```diff
@@ -228,7 +228,7 @@ func evalVariableValidations(addr addrs.AbsInputVariableInstance, config *config
 		diags = diags.Append(&hcl.Diagnostic{
 			Severity: hcl.DiagError,
 			Summary:  "No final value for variable",
-			Detail:   fmt.Sprintf("OpenTofu doesn't have a final value for %s during validation. This is a bug in Terraform; please report it!", addr),
+			Detail:   fmt.Sprintf("OpenTofu doesn't have a final value for %s during validation. This is a bug in OpenTofu; please report it!", addr),
```

Comment:
Oops, good catch.

Reply:
Thanks! `grep` has alway been enabling for "unexpected" discoveries 😄